docs/api-guide/views.md

```diff
@@ -217,11 +217,26 @@ You may pass `None` in order to exclude the view from schema generation.
     def view(request):
         return Response({"message": "Will not appear in schema!"})
 
+# Async Views
+
+When using Django 4.1 and above, REST framework allows you to work with async class based and function based views.
+
+For class based view, the view needs the handler methods such as `.get()`, `.post()`, `put()`, `patch()` and `.delete()` to be all async otherwise Django will raise an exception. For function based view, the view needs to be async.
+
+For example:
+
+    class ListUsers(APIView):
+        async def get(self, request):
+            return Response({"message": "This is an async class based view."})
+
+
+    @api_view(['GET'])
+    async def view(request):
+        return Response({"message": "This is an async function based view."})
```

Comment:
wrap in 

```python
code
```

Reply:
I'm using the same format as the rest of the documentation.

Reply:
Agreed with em1208. Current formatting should be prioritized for legacy repos, and changed later via refactoring if a decision project owners decide.